src/styles/main.scss

```diff
@@ -1,3 +1,38 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+}
+
+.box {
+  display: grid;
+  align-content: center;
+  justify-content: center;
+  grid-template-columns: repeat(3,minmax(300px,1fr));
```

Comment:
if you will use here **repeat(auto-fill, minmax(300px, 1fr))**  - then you could use just one @media for min-width 900px with 3 columns - but it's totally as you wish :) both decisions are working